core/dbt/contracts/graph/model_config.py

```diff
@@ -42,6 +42,20 @@ def _set_meta_value(obj: M, key: str, existing: Optional[Dict[str, Any]] = None)
     return result
 
 
+class OnConfigurationChangeOptions(str, Enum):
```

Comment:
I prefer the singular for a class name, so `OnConfigurationChangeOption`. Any objections?